cocos/renderer/core/gfx/GFXFence.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CC_CORE_GFX_FENCE_H_
+#define CC_CORE_GFX_FENCE_H_
```

Comment:
Can just use `#pragma once`

Reply:
I think [it's not C++ standard](https://en.wikipedia.org/wiki/Pragma_once), so I prefer it this way

Reply:
cocos2d-x uses it like this for years. Almost all compilers support it, and as i know, unreal use it too. It is ok you like this.